x/savings/client/cli/tx.go

```diff
@@ -42,7 +43,7 @@ func getCmdDeposit() *cobra.Command {
 		Use:   "deposit [amount]",
 		Short: "deposit coins to savings",
 		Example: fmt.Sprintf(
-			`%s tx %s savings 10000000bnb --from <key>`, version.AppName, types.ModuleName,
+			`%s tx %s deposit 10000000bnb --from <key>`, version.AppName, types.ModuleName,
```

Comment:
nit: I would add a multi-coins example

```
`%s tx %s deposit 10000000ukava,100000000usdx --from <key>`, version.AppName, types.ModuleName,

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/1197/commits/bdbb689d23e2a437d192736146a705c479a4ba50